crates/biome_cli/src/execute/process_file/lint.rs

```diff
@@ -31,6 +43,12 @@ pub(crate) fn lint_with_guard<'ctx>(
                     (Vec::new(), Vec::new())
                 };
             if let Some(fix_mode) = ctx.execution.as_fix_file_mode() {
+                let suppression_explanation = if suppress && suppression_reason.is_none() {
+                    "Ignored using `--suppress`".to_string()
```

Comment:
```suggestion
                    "ignored using `--suppress`".to_string()
```

This string is appended to a string that finished with `: `, so "ignored" should start with a lowercase letter. An upper-case letter isn't grammatically correct.